### DISCUSSION TITLE
为什么我的模组全变成英文了

### DISCUSSION THREAD

**Discussion by dmjxxwc** [Meloong-Git/PCL#1197] [提问] [answered]
我的问题吗为什么我的模组全变成英文了，英文渣两眼一黑啥模组是啥都看得懂了，找模组难度直线起飞
![QQ截图20230207131430](https://user-images.githubusercontent.com/124654845/217155341-fe1096e6-9630-41df-8a57-1dde4ade8dbb.png)

**Comment by WForst-Breeze** [comment, marked as answer]
可能是 #1175 导致的原因，请等待下一次紧急修复更新修复该问题

**Reply by WForst-Breeze** [reply]
Release/Snapshot 2.4.9 已修复

**Comment by dmjxxwc** [comment]
好的，能修复就行



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年2月7日(星期二) 中午1:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Hex-Dragon/PCL2] 为什么我的模组全变成英文了 (Discussion #1197)





 
可能是 #1175 导致的原因，请等待下一次紧急修复更新修复该问题

**Comment by dmjxxwc** [comment]
已更新，目前没有发现问题，多谢






------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年2月7日(星期二) 晚上11:25
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Hex-Dragon/PCL2] 为什么我的模组全变成英文了 (Discussion #1197)





 
Release/Snapshot 2.4.9 已修复
